src/main/webapp/resources/js/pages/password/ActivateAccount.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Alert, Button, Form, Input } from "antd";
+import { LoadingOutlined } from "@ant-design/icons";
+import { setBaseUrl } from "../../utilities/url-utilities";
+import { SPACE_MD } from "../../styles/spacing";
+import { useActivateAccountMutation } from "../../apis/password-reset";
+
+const { Item } = Form;
+
+/**
+ * React component to render the forgot password form
+ * @param {function} updateDisplayLoginPage Function to update whether to display login page
+ * @returns {*}
+ * @constructor
+ */
+export function ActivateAccount({ updateDisplayLoginPage }) {
+  const [loading, setLoading] = React.useState(false);
+  const [activateAccount] = useActivateAccountMutation();
+  const [activateAccountForm] = Form.useForm();
+  const [message, setMessage] = React.useState(null);
+  const activationIdRef = React.useRef();
+
+  /**
+   * When the component gets added to the page,
+   * focus on the activationId input.
+   */
+  React.useEffect(() => {
+    activationIdRef.current.focus();
+    activationIdRef.current.select();
+  }, []);
+
```

Comment:
The activation code is from the email.  Why don't we use the link in the email to autopopulate the input?